audio_context.html

```diff
@@ -0,0 +1,321 @@
+<!doctype html>
+<head>
+  <title>Web Audio Api - Audio Context</title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+
+<style>
+
+  :root {
+    --flex-direction: column;
+    --line-height: 4em;
+  }
+
+  @media(min-width: 250px) {
+    :root {
+      --flex-direction: row;
+      --line-height: 8em;
+    }
+  }
+
+  h1 {
+    text-align: center;
+  }
+
+  main {
+    outline: thick double;
+    background-color: whitesmoke;
+  }
+
+  fieldset {
+    display: flex;
+    border: none;
+    background: yellow;
+  }
+
+  form > fieldset {
+    flex-direction: var(--flex-direction);
+  }
+
+  fieldset fieldset {
+    background: red;
+  }
+
+  fieldset input[type="range"] {
+    width: 20vw;
+    display: inline-block;
+    top: 0.3em;
+    position: relative;
+  }
+
+  form > fieldset > label {
+    background: turquoise;
+    line-height: var(--line-height);
+  }
+
+  form > header {
+    background: yellow;
+  }
+
+  [type="checkbox"] {
+  }
+
+</style>
+
+</head>
+
+<body>
+
+  <header>
+    <h1>DevPunks Sequencer</h1>
+  </header>
+
+<main>
+
+  <form action=#>
+
+  <header>
+    DevPunks Sequencer
+
+    <label>
+      BPM
+      <input type=range min=10 max=522 value=251>
```

Comment:
What are your thoughts on this order for attributes.
```html
<input name=... value=... min=... max=... type=range >
```
Keeps the human interested stuff up front starting wirh variable name and value.  `type=...` more an implementation detail hence last. Jist a thought...

Reply:
I prefer this order, will change it around.